<?xml version="1.0"?>
<launch>
  <arg name="namespace" default="taluy" />
  <arg name="rate" default="20" />

  <group ns="$(arg namespace)">
    <node pkg="auv_localization" type="imu_odometry_node.py" name="imu_to_odom_node" output="screen">
      <remap from="sensors/imu/data" to="sensors/imu/data" />
      <remap from="localization/odom_imu" to="localization/odom_imu" />
      <param name="config_path" value="$(find auv_localization)/config/calibration_data.yaml"/>
    </node>
    <node pkg="auv_localization" type="pressure_odometry_node.py" name="pressure_to_odom_node" output="screen">
      <remap from="sensors/pressure/data" to="sensors/pressure/data" />
      <remap from="localization/odom_pressure" to="localization/odom_pressure" />
      <param name="config_path" value="$(find auv_localization)/config/calibration_data.yaml"/>
    </node>
  </group>
</launch>